EFCore.FirebirdSql/Internal/FbOptions.cs

```diff
@@ -57,9 +57,9 @@ private FbOptions GetSettings(string connectionString)
                     connection.Close();
                 }
             }
-            catch
+            catch (Exception ex)
             {
-                //
+                throw ex;
```

Comment:
This breaks the stack
You can remove the **throw ex;** and simplify for **throw;** only!